open_schools_platform/organization_management/circles/services.py

```diff
@@ -12,6 +12,7 @@ def create_circle(name: str, organization: Organization, description: str, capac
     if location is None:
         geolocator = Nominatim(user_agent="circles")
         coordinates = geolocator.geocode(address)
+        # If address is not recognized by Nominatim, then the following exception will appear
```

Comment:
it is not docstring)
make small documentation for this method with '''your documentation''' right after method name
describe cases when you don't define location but define adress, what would be if adress is wrong and etc